lib/gems/pending/VMwareWebService/MiqVimInventory.rb

```diff
@@ -2126,7 +2126,7 @@ def getMoProp_local(mo, path = nil)
     oc.MOR = oc.obj
     oc.delete('obj')
 
-    oc.propSet = [oc.propSet] unless oc.propSet.kind_of?(Array)
+    oc.propSet = Array.wrap(oc.propSet) unless oc.propSet.kind_of?(Array)
```

Comment:
Being picky, but are you depending on the differences of `Array.wrap` here or will `Array(oc.propSet)` suffice? (don't use AS unless you really need to)